components/omega/doc/design/VertCoord.md

```diff
@@ -0,0 +1,292 @@
+(omega-design-vert-coord)=
+# Vertical Coordinate
+
+## 1 Overview
+The vertical coordinate module will be responsible for computing and storing information related to the vertical mesh in Omega.
+Since Omega will be a non-Boussinesq model, the vertical independent variable will be pressure- as opposed to $z$-based.
+Similar to MPAS-Ocean's support for tilted height ($z^\star$) coordinates, Omega will support a general ($p^\star$) vertical coordinate.
```

Comment:
We can actually do either, since there is an easy computation between $z$ and $p$ and both are available.

```suggestion
Similar to MPAS-Ocean, the vertical coordinate will be Arbitrary Lagrangian-Eularian (ALE). This means that the vertical layer interfaces will be computed within the ALE routine, and may be chosen to be z-star ($z^\star$, with layer thickness varying in proportional to sea surface height in $z$) or p-star ($p^\star$, varying in proportion to bottom pressure, in $p$). P-star is a more natural choice for a non-Boussinesq model.
```

Reply:
> varies in proportion to the bottom pressure.

It that correct?  I thought it would vary proportion to the total mass-thickness of the water column.